connector/src/main/scala/com/basho/riak/spark/rdd/RiakTSRDD.scala

```diff
@@ -151,6 +133,14 @@ class RiakTSRDD[R] private[spark](
   def sql(query: String): RiakTSRDD[R] = {
     copy(query = Some(query))
   }
+  
+  def filter(filters: Array[Filter]): RiakTSRDD[R] = {
```

Comment:
can we limit the visibility of this method, since it is expected that it will be used under the hood of Spark Connector